app/src/main/java/ch/epfl/skysync/components/ChatText.kt

```diff
@@ -38,6 +36,7 @@ import androidx.compose.ui.Modifier
 import androidx.compose.ui.graphics.Color
 import androidx.compose.ui.platform.LocalSoftwareKeyboardController
 import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.text.TextStyle
 import androidx.compose.ui.text.input.ImeAction
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
```

Comment:
I can't add comment to line 84 because it is old code but: The `LaunchedEffect` should use `messages` as a key to ensure the list scrolls to the latest message whenever `messages` change. Ex: ```LaunchedEffect(messages) {
        if (messages.isNotEmpty()) {
            lazyListState.scrollToItem(messages.size - 1)
        }
    }```